ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -1147,7 +1147,17 @@ const triggerSet: TriggerSet<Data> = {
       condition: (data) => Conditions.targetIsYou() && data.act === 'curtain',
       delaySeconds: (data, matches) => {
         const duration = parseFloat(matches.duration);
-        return (data.role === 'dps' ? duration - 12 : duration - 6);
+
+        // Check if custom duration was set in options
+        let breakTime = parseFloat(typeof data.options.cactbotp4sCurtainCallBreakTime === 'string' ? data.options.cactbotp4sCurtainCallBreakTime : '6' );
```

Comment:
I know there were a couple of options previously, but I think I would really like to move to just "strat specific things belong in user triggers" and not try to support a pile of options.

To me, it would make sense to just have the debuff call out Group 1 / Group 2 / Group 3 / Group 4, and then folks can figure it out from there.

Reply:
I think this can be done. I currently have it waiting to call out until 12s on the debuff and changed to infoText as the player may or may not need to do something right away.

It could call out as soon as debuffs went out (or in addition). I chose not to call out at start because it would duplicate the callout (although we could handle it such as only groups 2-4 or 3/4 get a reminder), and also did not want to leave the text up for too long (42s at the worst).

Reply:
For a break tether callout at 6 for better or worse if a callout were implemented I'd want it to check that the debuff is still on the player so it's not calling it out when the player already broke the tether... would that necessitate OverlayHandler?

Reply:
I think it's better to call out right at the beginning so people know.  Partially it's that group 1 needs to be ready to go (they could get an alarm too?), and also that the tanks need to know if they need to tank swap too.  Personally, I think the "when to break" is less necessary and a more damage dependent timing?  I think people will be able to break faster as gear improves too.

I am not sure how to figure out if a tether has broken on a particular player or not.

Reply:
There is a limit to how fast the tethers can be broken due to the vulnerability up that occurs, which we could actually use to possibly indicate a break tether during the groups for the second person to know they could break.

Reply:
Oh I see how we can use that magic vulnerability up even more. We just have a counter that counts them, and just say break tether to the person next in line once the previous is gone.

Reply:
Hmmm, looking at it, I think the call breaks one I wrote may best to just call when group 2, 3, 4 can start than to call their break

Reply:
One note on adding a tank swap... would need to know if the data.role has aggro of the boss which I am not sure at the moment where to get that info.

Reply:
Quick question for Curtain Call, how difficult would it be to have a trigger that would call the order of the breaks using party members' names? I feel like that's a trigger that would help groups stay afloat during that mechanic. Or perhaps is it too much info?

Reply:
As it is currently, each group gets there own tailored call out instead of everyone receiving 4 callouts. It's possible to add the names of each player to the groups, but i think this would require an additional trigger for collecting those names and then adding a tiny delay for the collection to complete. I do not see a whole lot of benefit to this unless you were to be running some non-standard comp where you couldn't get the role info or you perhaps had deaths and an uneven amount of debuffs went out.

The later check of which group is next also could go out to everyone as an infotext instead of the individual players. There wouldn't need to be an extra trigger for there as we could just use the extra trigger created by the initial check. Adding the names to the group 2-4 calls is the same.

Reply:
From a raid leader stand point, I could see that knowing which players are to break would help  but this is going to be a lot of info.

Reply:
Yeah, I think that sounds like a personal trigger to me as well.